### ISSUE TITLE
Stars

### ISSUE THREAD

**Issue by techtonik** [jayfk/fundingoss.com#9]
Would be nice to star projects..

**Comment by pydanny** [comment]
I think a combination of voting process (+1's) combined with the number of GitHub stars would be awesome. This would give both established projects and unlaunched inventive projects that could solve problems better representation on the site.

**Comment by ralphtheninja** [comment]
You could also have something like http://node-modules.com/, where the order is based on your social graph on github, if you login with your github account that is :)

**Comment by jayfk** [comment]
What if we added a voting system where a user can vote for the importance of the project, from 1 to 100? 

As much as I like, say `django-debug-toolbar`, the project is not that important to the whole eco system as something like `openssl`, or `freebsd` or `apache`.

It'd nice if we could find a way to reflect that not only by the count of votes in total.

What do you think?

**Comment by techtonik** [comment]
> What if we added a voting system where a user can vote for the importance of the project, from 1 to 100?

@jayfk we had that at Gratipay, where people naturally set their weekly voting budget in $$$ into as little as $0.01 chunks per project. I never got to building a graph out of it, but we still have that info. If @whit537 manages to get query to fetch table of most 'voted' projects in https://github.com/gratipay/inside.gratipay.com/issues/471#issuecomment-173130516 then we can think about how to fetch depersonalized graph of that for an example of the idea.

**Comment by heyakyra** [comment]
Doesn't this open up a can of worms wrt populism and vote brigades?

**Comment by pydanny** [comment]
Yes it does open up a nasty can of worms. That's why limiting it to just +1 means a simpler, hence hopefully more easily detected for vote mongering system.

That said your point is strong enough that it does worry me.

**Comment by nayafia** [comment]
I think adding number of GitHub stars is a great start. I am hesitant about user voting b/c two people can vote for a project for different reasons (one might think the theoretical potential is high, another might think current dependencies matter more), which won't be reflected in pure votes unless we provide more direction.

I do think figuring out how to rank projects is important. I noodled on this last month and Linux CII has some interesting criteria, going to start a separate issue around this.
